vips/vip-453/permissions.ts

```diff
@@ -0,0 +1,88 @@
+import ACM_COMMANDS_AGGREATOR_ABI from "@venusprotocol/governance-contracts/artifacts/contracts/Utils/ACMCommandsAggregator.sol/ACMCommandsAggregator.json";
+import { ACMCommandsAggregator } from "@venusprotocol/governance-contracts/typechain/contracts/Utils/ACMCommandsAggregator";
+import { ethers } from "hardhat";
+import hre from "hardhat";
+import { AccountType, getXVSBridgeAdminPermissions, getXVSPermissions, getXVSVaultPermissions } from "src/permissions";
+
+interface Permissions {
+  [key: string]: string[][];
+}
+
+const BERACHAINBARTIO_GUARDIAN = "0xdf3b635d2b535f906BB02abb22AED71346E36a00";
+const XVS_BRIDGE_ADMIN = "0xdE489177E607F1C6D9d27325FA38152fA462F7cC";
+const XVS = "0x75A3668f0b0d06E45601C883b0c66f7Dd2364208";
+const XVS_VAULT = "0x622E4e8104F7ebc94Aa81Db2613A751c39e6688b";
+
+const grantPermissions: Permissions = {
+  berachainbartio: [
+    ...getXVSBridgeAdminPermissions(XVS_BRIDGE_ADMIN),
+    ...getXVSPermissions(XVS),
+    ...getXVSVaultPermissions(XVS_VAULT),
+  ],
+};
+
+const acmCommandsAggreator: any = {
+  berachainbartio: "0x1ba10ca9a744131aD8428D719767816A693c3b71",
+};
+
+const accounts: any = {
+  berachainbartio: {
+    NormalTimelock: "0x8699D418D8bae5CFdc566E4fce897B08bd9B03B0",
+    FastTrackTimelock: "0x723b7CB226d86bd89638ec77936463453a46C656",
+    CriticalTimelock: "0x920eeE8A5581e80Ca9C47CbF11B7A6cDB30204BD",
+    Guardian: BERACHAINBARTIO_GUARDIAN,
+  },
+};
+
+function splitPermissions(
```

Comment:
It seems we need to configure 93 permissions for this VIP, so we don't need to split them into different chunks. I would remote this logic from this specific file. If we need it in the future, we could add it to the `src/permissions.ts` file.

This VIP is specific for `berachainbartio`, we can assume it, and remove the network dimension  from the different variables in this file (`berachainbartio`), to simplify it.